mirrord/layer/src/lib.rs

```diff
@@ -141,36 +142,53 @@ pub(crate) fn port_debug_patch(addr: SocketAddr) -> bool {
     }
 }
 
+/// Loads mirrord configuration and applies [`nix_devbox_patch`] patches.
+fn layer_pre_initialization() -> Result<(), LayerError> {
+    let args = std::env::args().collect::<Vec<_>>();
+
+    let given_process = args
+        .first()
+        .and_then(|arg| arg.split('/').last())
+        .ok_or(LayerError::NoProcessFound)?;
+
+    let mut config = std::env::var("MIRRORD_CONFIG_FILE")
+        .map_err(LayerError::from)
+        .map(PathBuf::from)
+        .and_then(|path| Ok(LayerFileConfig::from_path(&path)?))
+        .unwrap_or_default()
+        .generate_config()?;
+
+    nix_devbox_patch(&mut config);
+    let skip_processes = config.skip_processes.clone().map(VecOrSingle::to_vec);
```

Comment:
maybe we can remove the nixos patch in this PR, see https://github.com/metalbear-co/mirrord/issues/711

Reply:
How do I test this?

Reply:
I opened it after I saw this PR and thought that we can remove it (irrelevant to this PR though, we fixed the root cause in another one)
I don't think it should be fixed as part of it. Also it's less prioritized